docs/scheduledtasks.md

```diff
@@ -0,0 +1,25 @@
+### Scheduled Tasks
+The Framework is responsible for executing all Open vStorage scheduled tasks. These tasks are kicked of by celery beat, the task scheduler of Celery. To schedule the exact timing of these tasks, celery schedules are used. These schedules are store in the [distributed configuration management system](configmgmt.md).  
+
+An optional new key is introduced in the Configuration management: `/ovs/framework/scheduling/celery` which contains a JSON dictionary where the key is the task's name, and the value is:
+
+* null in case the task should be disabled (not automatically executed)
+* A dict containing [crontab[(http://docs.celeryproject.org/en/latest/reference/celery.schedules.html#celery.schedules.crontab)] keyword arguments
+
+Example configuration:
+
+```
+{
+    "ovs.scheduled.collapse_arakoon": {"minute": "0", "hour": "*"},
```

Comment:
Please use other examples, unless the collapse task is explained. You can better use ```ovs.scheduled.execute_scrub``` and ```alba.scheduled.verify_namespaces``` instead, as this one is explained below.